Test-NetStack.psm1

```diff
@@ -156,7 +156,14 @@ Function Test-NetStack {
         [Parameter(Mandatory = $false, ParameterSetName = 'OnlyPrereqIPTarget'   , position = 4)]
         [Parameter(Mandatory = $false, ParameterSetName = 'RevokeFWRulesNodes'   , position = 4)]
         [Parameter(Mandatory = $false, ParameterSetName = 'RevokeFWRulesIPTarget', position = 4)]
+
         [Switch] $Experimental = $false,
+        [Parameter(Mandatory = $false)]
+        [String] $DpdkUser='',
+        [Parameter(Mandatory = $false)]
+        [String[]] $DpdkPortIps ='',
+	    [Parameter(Mandatory = $false)]
```

Comment:
I don't know what's going on here, tried fixing it both in editor and on the web and nothing seems to work. 👎